test/e2e/clusterclass_rollout.go

```diff
@@ -279,7 +279,7 @@ func assertClusterObjects(ctx context.Context, clusterProxy framework.ClusterPro
 		assertMachineSetsMachines(g, clusterObjects, cluster)
 
 		By("All cluster objects have the right labels, annotations and selectors")
-	}, 10*time.Second, 1*time.Second).Should(Succeed())
+	}, 30*time.Second, 1*time.Second).Should(Succeed())
```

Comment:
@sbueringer do you know of any changes recently that could have caused this? The flake only popped up on June 16.

Reply:
But totally different controller